homeassistant/components/esphome/cover.py

```diff
@@ -125,8 +125,9 @@ async def async_close_cover_tilt(self, **kwargs: Any) -> None:
         """Close the cover tilt."""
         await self._client.cover_command(key=self._static_info.key, tilt=0.0)
 
-    async def async_set_cover_tilt_position(self, **kwargs: int) -> None:
+    async def async_set_cover_tilt_position(self, **kwargs: Any) -> None:
```

Comment:
Currently the async_set_cover_tilt_position only has one argument (ATTR_TILT_POSITION: vol.Coerce(int)), but I still think it is better to keep this method future-proof and use `Any` for the `**kwargs`.
This also keeps it consistent with the implementation in all other components.